templates/pages/signup.jsx

```diff
@@ -87,7 +87,7 @@ var Signup = React.createClass({
           <IconText iconClass="agreement" textClass="eula">
             By signing up, I agree to Webmaker&lsquo;s <a tabIndex="5" href="//beta.webmaker.org/#/legal" className="underline">Terms of Service</a> and <a tabIndex="6" href="//www.mozilla.org/privacy/websites" className="underline">Privacy Policy</a>.
```

Comment:
Yeah, I didn't want to change that in case it had an legal repercussions. Agreed that it's a good follow-up issue.